.pdkignore

```diff
@@ -0,0 +1,47 @@
+.git/
```

Comment:
The `.pdkignore` is generated from https://github.com/puppetlabs/pdk-templates/blob/main/moduleroot/.pdkignore.erb with the default configuration of https://github.com/puppetlabs/pdk-templates/blob/main/config_defaults.yml

It might be easier, right. And I am also not really sure why the PDK templates does the things it does. I just applied it. But if you want we can investigate and document everything in more detail.